automat/_test/test_visualize.py

```diff
@@ -1,16 +1,34 @@
-
-from __future__ import unicode_literals
+import functools
 
 import os
 import subprocess
 from unittest import TestCase, skipIf
 
+from characteristic import attributes
+
 from .._methodical import MethodicalMachine
 
+from .._visualize import elementMaker, tableMaker
+
+
+def isGraphvizModuleInstalled():
+    """
+    Is the graphviz Python module installed?
+    """
+    try:
+        import graphviz
+    except ImportError:
+        return False
+    else:
+        del graphviz
```

Comment:
This was a silly attempt to quiet pyflakes.  I didn't want to write `__import__('graphviz')`, which would allow me not to bind a name at all at the expense of clarity.  Note that this function is also responsible for the decrease in coverage.  These two things make me feel that I'm missing some more elegant approach.

I'll gladly remove the `del` noise, though!